lib/commands/install.sh

```diff
@@ -37,21 +37,35 @@ get_concurrency() {
 }
 
 install_local_tool_versions() {
-  local asdf_versions_path
-  asdf_versions_path=$(find_tool_versions)
-  if [ -f "${asdf_versions_path}" ]; then
-    while IFS= read -r tool_line; do
-      IFS=' ' read -r -a tool_info <<< "$tool_line"
-      local tool_name
-      tool_name=$(echo "${tool_info[0]}" | xargs)
-      local tool_version
-      tool_version=$(echo "${tool_info[1]}" | xargs)
-
-      if ! [[ -z "$tool_name" || -z "$tool_version" ]]; then
-        install_tool_version "$tool_name" "$tool_version"
+  local plugins_path
+  plugins_path=$(get_plugin_path)
+
+  local search_path
+  search_path=$(pwd)
+
+  local some_tools_installed
+
+  if ls "$plugins_path" &> /dev/null; then
```

Comment:
If you are just wanting to check if this directory exists, `[ -d "$plugins_path" ]` is going to work on more systems. If you are wanting to check for subdirectories I think you will need `ls "$plugins_path"/*/`.

Reply:
I've taken this code as-is from 
https://github.com/asdf-vm/asdf/blob/master/lib/commands/plugin-list.sh#L11, so perhaps it's better to change it in both places?

Reply:
Yes, but I suppose I can merge these changes as they are. We always update this later.